linode/manager
# feat: [M3-8016] - Add TagSelect to edit images drawer

## Description 📝
Adds a TagSelect to the edit image drawer based on the new field in the API.

## Changes  🔄
- Add TagSelect to edit image drawer
- Update APIv4 Image type and factory with new `tags` field
- Clean up `ImageDrawer` component, removing unused code

## Target release date 🗓️
5/28

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-05-14 at 1 52 31 PM](https://github.com/linode/manager/assets/122488130/d57e654b-b825-41f5-bd78-e3925565b7be) | ![Screenshot 2024-05-14 at 1 52 19 PM](https://github.com/linode/manager/assets/122488130/eec2c037-2923-4ff7-8259-d10834cf4c7c) |

## How to test 🧪

### Prerequisites
- Configure Cloud Manager to use the alpha API (e.g., via DevTools)

### Verification steps
- Navigate to the Images page
- Click on the Action menu for an image and select 'Edit'
- Verify tag select appears
- Add/remove tags and click "Save Changes"
- Re-open edit drawer and verify changes are reflected
- Verify that there are no regressions in the Restore from Image drawer, which reuses this component

## TODO
- [x] Create unit tests for ImageDrawer

## As an Author I have considered 🤔

*Check all that apply*
